### ISSUE TITLE
environment folder hierachy

### ISSUE THREAD

**Issue by R-Sweke** [HendrikPN/scigym#4]
As the package grows there will hopefully be many more environments added. At present, all environments are subdirectories of the directory `scigym/envs/`. Would it not make sense to categorize the environments (at least in a very coarse grained way) such that the package structure mirrors any  categorization that might be used on the website - i.e. into broad categories such as physics, biology, chemistry etc?
 
For instance, perhaps "environment j" in "category i" should be housed in the following subdirectory:

`scigym/envs/category_i/env_j/`

If this would be useful, what would be natural categorizations?

**Comment by HendrikPN** [comment]
Ah, I didn't see this until now. Yes, very good point! Ideally, this would also be in line with the structure on our homepage!
Still, where do we stop: The homepage, at this point, allows sub-subcategories, e.g.
```
scigym/envs
|----quantum_physics
      |----quantum_computing
      |----quantum_information
      |----quantum_foundations
```
which would make for a long name: `scigym/envs/quantum_physics/quantum_computing/surfacecode_decoding`. I mean, I am pretty fine with that. What do you think?

**Comment by R-Sweke** [comment]
I agree with more structure, at the expense of verbosity, and I think your suggestion for categorizations is perfect. 

Exactly this directory hierachy has now been included in commit  [e0b0f91](/HendrikPN/scigym/pull/1/commits/e0b0f91708ea6e269e39a9ffffee4a26f06f0af3) of pull request #1.

**Comment by HendrikPN** [comment]
PR #1 closes this issue
